### ISSUE TITLE
ALOpsLicenseImport help needed

### ISSUE THREAD

**Issue by mbaic** [HodorNV/ALOps#21]
Hi,

I am trying to make this task work:
![image](https://user-images.githubusercontent.com/35611941/62278848-2ea9c100-b449-11e9-8350-75dce825e7d8.png)
Sadly, not able to succeed till now. Error that I'm getting below is in red:
![image](https://user-images.githubusercontent.com/35611941/62278924-57ca5180-b449-11e9-9ab3-5f4b9cbb425e.png)
Definition in YAML is:
![image](https://user-images.githubusercontent.com/35611941/62279365-387ff400-b44a-11e9-86fb-3d920e5e57c2.png)
I am using CopyFiles task to prepare a license path for ALOpsLicenseImport task and output is:
![image](https://user-images.githubusercontent.com/35611941/62279591-b217e200-b44a-11e9-8e32-fbfe48f3f9a4.png)

Could you please help/advice me how to use a *.flf file in described scenario above where agent is "Hosted VS2017" or so, not self hosted, and to import it in to the docker container via ALOpsLicenseImport task? I would not use a downloadable Url as an option.

At this very moment, I am just trying to test these "ALOps" tasks, that's why I don't use self hosted agent.

Many thanks in advance.

**Comment by mbaic** [comment]
Hi again,

In the meantime, I've succeeded with VM and self hosted agent. But, I am still curious how to handle license issue described above, in previous comment.

Any response will be appreciated.

**Comment by waldo1001** [comment]
I notice you are including your license in the repository.  Try to refer it with a relative path, in that case, like this example:
```
- task: ALOpsLicenseImport@1
    displayName: 'Repository license file'
    inputs:
      usedocker: true
      license_path: '.\license\license.flf'
```

**Comment by mbaic** [comment]
Hi again,

Tried that option as well:
![image](https://user-images.githubusercontent.com/35611941/62519740-7c00a680-b82c-11e9-9a55-0158375b1a10.png)
In this scenario with Microsoft hosted agent 'Hosted VS2017' and ALOps, it seems a problem is not in source path for the license file. The problem is coping to the target docker container path where drive D does not exist. Am I right? 

Any further suggestions on how to handle this will be appreciated.

Thank you in advance for your time.

**Comment by waldo1001** [comment]
Well, it's again on a hosted option .. we'll have to look into that as well .. at this point, all simulations we do with license files (downloaded, part of repo, or part of build server) works on our own machines.. .

we'll keep you posted!

**Comment by AdminHodor** [comment]
Fix for this will be available in the next release v1.423
